src/api/spec/features/webui/notifications_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "browser_helper"
+
+RSpec.feature 'Notifications', :type => :feature, :js => true do
+  let(:admin_user) { create(:admin_user) }
+
+  scenario 'update notifications' do
+    login admin_user
+    visit notifications_path
+
+    page.has_content?('Events to get email for')
+
+    checkboxes = %w(
+      Event::CommentForPackage_commenter
+      Event::CommentForProject_maintainer
+      Event::CommentForRequest_reviewer
+      Event::BuildFail_maintainer
+    )
+    checkboxes.each do |checkbox|
+      check(checkbox)
+    end
+
+    click_button 'Update'
+    page.has_content?('Notifications settings updated')
```

Comment:
Same here, should be:
Why this? I mean, it only returns true or false but don't expect's nothing. Did you mean something like this? 
```
expect(page).to have_content('Notifications settings updated')
```